src/components/canvas-tools/geometry-tool.tsx

```diff
@@ -205,4 +205,21 @@ class GeometryToolComponent extends BaseComponent<IProps, IState> {
   }
 }
 
-export default sizeMe()(GeometryToolComponent);
+export default class GeometryToolComponent extends React.Component<IProps, {}> {
```

Comment:
We have to switch from the `sizeMe` HOC to the explicit second class using the <SizeMe> component because the static `getDragImageNode()` function has to be in the wrapper class, but there's no way to specify that with the HOC.